### DISCUSSION TITLE
React does not recognize the  disablePlaceholder prop on a DOM element.

### DISCUSSION THREAD

**Discussion by jahnavijanu143** [vercel/next.js#29337] [Help] [answered]
### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

v14.17.3

### What browser are you using?

chrome

### What operating system are you using?

windows

### How are you deploying your application?

aws,docker

### Describe the Bug

Warning: React does not recognize the `disablePlaceholder` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `disableplaceholder` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

### Expected Behavior

Run image tag without showing warning

### To Reproduce

<Image
            src={image2}
            alt={picture}
            width={88}
            height={88}
            //placeholder="blur"
            unoptimized={true}
            disablePlaceholder={true}
           />
This are the properties i used for image tag
Do i need to add any properties?
I  have a question, this  disablePlaceholder attribute will disable the placeholder?

**Comment by ijjk** [comment, marked as answer]
Hi, `disablePlaceholder` is not a prop for `next/image` and the placeholder is only enabled when set so can be removed. See list of `next/image` props here https://nextjs.org/docs/api-reference/next/image
